lib/build.gradle

```
@@ -0,0 +1,13 @@
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
```

```suggestion
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8
```
자바 버전을 1.8로 사용해주시는 것을 권장합니다.

---

1.8 버전으로 수정하였습니다!